config/perf_configs/jet_n_failures.yml

```diff
@@ -1 +1,2 @@
 job_id: "jet_n_failures"
+implicit_diffusion: false
```

Comment:
@szy21, why was this added to this jet job?

Reply:
Oh, if it's not clear, `implicit_diffusion` has been always false in this jet job. But I set it to true in the default perf config in this PR, so this line is to set it to false again.